assets/policy.rego

```diff
@@ -120,6 +120,7 @@ tags(extra_tags) = array.concat([tag | tag := extra_tags[_]; contains(tag, ":")]
 	sprintf("drift_detection:%s", [input.run_updated.run.drift_detection]),
 	sprintf("run_note:%s", [input.run_updated.note]),
 	sprintf("run_type:%s", [lower(input.run_updated.run.type)]),
+    sprintf("run_url:%s", [lower(input.run_updated.urls.run)]),
```

Comment:
`lower` should be removed, run ID use uppercase letters - for example: `https://<subdomain>.app.spacelift.dev/stack/dummy/run/01J82C3Q7V4ASKKQ7DAY0MJR6D`